src/mcts/search.cc

```diff
@@ -1281,18 +1277,44 @@ void SearchWorker::DoBackupUpdateSingleNode(
     return;
   }
 
+  // For the first visit to a terminal, maybe convert ancestors to terminal too.
+  auto can_convert = params_.GetTerminator() && node->IsTerminal() && !node->GetN();
```

Comment:
Line longer than 80 characters, run `clang-format -style=google` (or try to fix that manually as we still don't have merge check for that)

Reply:
Added newline for now (as depending on params decision, this might change again).